docs/pages/accounts/audit-logs.mdx

```diff
@@ -39,30 +39,42 @@ The Apple devices listed within the Expo team's settings will only show devices
 
 While we are working on adding more entities in future, the following entities are already enabled:
 
-- Accounts
-- Android App Credentials
+- Account
```

Comment:
Hi @tchayen, I was going through dashboard changes and found we list more entities now under Audit logs. Is it okay to reflect the same info here in the doc?